### PR TITLE
[GLOSSARY] block

### PR DESCRIPTION
**Pull request by MaxHillebrand**
adds block to glossary

### CODE REVIEW

File: docs/glossary/Glossary.md [hunk 1]
```diff
@@ -20,6 +20,15 @@ A design document for introducing features or information to Bitcoin.
 Read more: [Wasabi Supported BIPs](/using-wasabi/BIPs.md)
 :::
 
+:::details
+### Block
+
+A block is a batch of valid Bitcoin transactions and it must hash to a value below a certain difficulty target to proof the work of the miner.
```
Comment (SimoneBovi) [inline docs/glossary/Glossary.md]: A block is a batch of valid Bitcoin transactions and its hash must be a value below a certain difficulty target to proof the work of the miner.

Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]:
```suggestion
A block is a batch of valid Bitcoin transactions and its hash must be a value below a certain difficulty target to proof the work of the miner.
```

**Comment by RiccardoMasutti** [comment]
NACK.
I think Bitcoin basics should not be in Wasabi glossary.

File: docs/glossary/Glossary.md [hunk 2]
```diff
@@ -20,6 +20,15 @@ A design document for introducing features or information to Bitcoin.
 Read more: [Wasabi Supported BIPs](/using-wasabi/BIPs.md)
 :::
 
+:::details
+### Block
+
+A block is a batch of valid Bitcoin transactions and its hash must be a value below a certain difficulty target to proof the work of the miner.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]:
```suggestion
A block is a batch of valid Bitcoin transactions and its hash must be a value below a certain difficulty target to prove the work of the miner.
```

File: docs/glossary/Glossary.md [hunk 3]
```diff
@@ -20,6 +20,15 @@ A design document for introducing features or information to Bitcoin.
...
+Each block references the hash of the previous block, thus creating a hash chain of blocks.
+On average a new block is created every 10 minutes.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]:
```suggestion
On average a new block is mined (created) every 10 minutes.
```

File: docs/glossary/Glossary.md [hunk 4]
```diff
@@ -20,6 +20,15 @@ A design document for introducing features or information to Bitcoin.
...
+A block can be up to 4 weight units [or 4 virtual bytes] in size.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]: Maybe we should remove this because the size of the block may change.

File: docs/glossary/Glossary.md (hunk 3 above)
Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]:
```suggestion
On average a new block is mined every 10 minutes.
```

File: docs/glossary/Glossary.md (hunk 4 above)
Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]:
```suggestion
```

**Comment by MaxHillebrand** [comment]
`Block` is mentioned 19 times in the docs, thus I think it should definitely be included.

**Pull request merged**